gemma-core/src/main/java/ubic/gemma/persistence/service/common/description/CharacteristicService.java

```diff
@@ -95,6 +95,10 @@ public interface CharacteristicService extends BaseVoEnabledService<Characterist
      */
     Collection<Characteristic> findByValue( String search );
 
+    Map<String, CharacteristicDao.CharacteristicByValueUriOrValueCount> countCharacteristicValueLikeByValueUriOrValue( String search );
```

Comment:
Rename suffix to `ByNormalizedValue` which stands for either `valueUri` or `value` if the former is null or empty.